include/psen_scan_v2/reply_msg_from_scanner.h

```diff
@@ -41,23 +41,30 @@ class ReplyMsgFromScanner
 public:
   static ReplyMsgFromScanner fromRawData(const RawScannerData& data);
 
-public:
-  ReplyMsgFromScanner(const uint32_t op_code, const uint32_t res_code);
+private:
```

Comment:
Please, try to stick to a certain order of the access modifiers for easier readability:
1. First, all public methods. This is the most important point. This allows the reader to overview the interface of the class quickly. 
2. Then all private methods.
3. Then all private member, etc.